src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+import { PaginationOption } from '../../types/PaginationOption';
+import { getPages } from '../../services/utils';
+import { Page } from '../Page';
+
+type Props = {
+  paginationOption: PaginationOption,
+  onPageChange: (value: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  paginationOption,
+  onPageChange,
+}) => {
```

Comment:
Fields with paginationOption are often reused, so it is worth destructuring this object.